sklearn/metrics/scorer.py

```diff
@@ -229,8 +251,8 @@ def get_scorer(scoring):
             scorer = SCORERS[scoring]
         except KeyError:
             raise ValueError('%r is not a valid scoring value. '
-                             'Use sorted(sklearn.metrics.SCORERS.keys()) '
-                             'to get valid options.' % (scoring))
+                             'Use sorted(sklearn.metrics.valid_scorers()) '
```

Comment:
But the message here should remain the same right?

Reply:
hm you're right, that's a bit weird if we make it private.
We could just print the output of ``valid_scorers`` here. I think we removed this here at some point but I don't remember why.

Reply:
https://github.com/scikit-learn/scikit-learn/pull/14123#issuecomment-510043140

I can't remember either. Maybe I thought the list will be too long for an error message. Now, I believe that's the easiest way and I will get rid of the valid_scorers method too

Reply:
Now I know why I decided to do that. We print the output of a wrong call in the documentation. https://github.com/scikit-learn/scikit-learn/pull/14123/files#diff-1a86c825aeb118f98447f3435891d362R112.
When adding a new scorer we should update the documentation every time which is not explicit at all. What do you think about going back to valid_scorers function?